eslint-plugin-prettier.js

```diff
@@ -45,15 +45,45 @@ let prettier;
  * @returns {Object} An object containing numeric `line` and `column` keys.
  */
 function getLocFromIndex(context, index) {
-  // If `sourceCode.getLocFromIndex` is available from ESLint, use it.
-  // Otherwise, use the private version from eslint/lib/ast-utils.
-  // `sourceCode.getLocFromIndex` was added in ESLint 3.16.0.
+  // If `sourceCode.getLocFromIndex` is available from ESLint, use it - added
+  // in ESLint 3.16.0.
   const sourceCode = context.getSourceCode();
   if (typeof sourceCode.getLocFromIndex === 'function') {
     return sourceCode.getLocFromIndex(index);
   }
+  // Otherwise, try to use the private version from eslint/lib/ast-utils -
+  // added in ESLint 3.11.0.
   const astUtils = require('eslint/lib/ast-utils');
```

Comment:
I haven't figured out the root cause yet, but I think using `require('eslint')` might be the source of #11 and #19. That could happen if the module found by running `require('eslint')` is not the same as the eslint module that is actually running the rule, which could result in neither `sourceCode.getLocFromIndex` nor `astUtils.getLocationFromRangeIndex` existing even when both modules are above 3.11.0.

That said, I guess this isn't a problem anymore because the fallback will get used now if neither function exists.